docs/en/latest/plugins/dubbo-proxy.md

```diff
@@ -100,6 +100,31 @@ curl http://127.0.0.1:9080/apisix/admin/routes/1  -H 'X-API-KEY: edd1c9f034335f1
 You can follow the [Quick Start](https://github.com/alibaba/tengine/tree/master/modules/mod_dubbo#quick-start) example in Tengine and use the configuration above to test it.
 They should provide the same result.
 
+The returned data from upstream dubbo service must be a `Map<String, String>`.
+
+If the returned data is
+
+```json
+{
+    "status": "200",
+    "header1": "value1",
+    "header2": "valu2",
+    "body": "blahblah"
```

Comment:
Does array type data exist here? If so, I think we should add one example.

Reply:
No. Only string can be used.

Reply:
Got it.